docker_entrypoint.sh

```diff
@@ -9,48 +9,60 @@ _term() {
 echo "Starting Container..."
 TOR_ADDRESS=$(yq e '.tor-address' /root/start9/config.yaml)
 LAN_ADDRESS=$(yq e '.lan-address' /root/start9/config.yaml)
-CONNECTION=$(yq e '.enable-tor' /root/start9/config.yaml)
 SERVICE_ADDRESS='nextcloud.embassy'
-NEXTCLOUD_ADMIN_USER=$(yq e '.username' /root/start9/config.yaml)
-NEXTCLOUD_ADMIN_PASSWORD=$(yq e '.password' /root/start9/config.yaml)
+NEXTCLOUD_ADMIN_USER='embassy'
 POSTGRES_DATADIR="/var/lib/postgresql/13"
 POSTGRES_CONFIG="/etc/postgresql/13"
 NEXTCLOUD_TRUSTED_DOMAINS="$TOR_ADDRESS $LAN_ADDRESS $SERVICE_ADDRESS"
 TRUSTED_PROXIES="$TOR_ADDRESS $LAN_ADDRESS $SERVICE_ADDRESS"
```

Comment:
I don't see NEXTCLOUD_TRUSTED_DOMAINS and TRUSTED_PROXIES being used internally. Verify if this is working, bash might be more forgiving but normally this should be exported.

Reply:
yes

```

2023-02-06T18:04:46-05:00  Nextcloud was successfully installed
2023-02-06T18:04:46-05:00  Setting trusted domains…
2023-02-06T18:04:47-05:00  System config value trusted_domains => 1 set to string f2hbleftsndxaitsfju72ynyrwzho4si3a5vutvkwvsktfhykbxmk5id.onion
2023-02-06T18:04:48-05:00  System config value trusted_domains => 2 set to string f2hbleftsndxaitsfju72ynyrwzho4si3a5vutvkwvsktfhykbxmk5id.local
2023-02-06T18:04:49-05:00  System config value trusted_domains => 3 set to string nextcloud.embassy
2023-02-06T18:04:49-05:00  Initializing finished
```